pkg/time/time.go

```diff
@@ -52,7 +56,7 @@ func add(l *lua.State) int {
 	hour := lua.OptInteger(l, -3, 0)
 	l.Pop(3)
 
-	inc := startUnix.Add(time.Hour * time.Duration(hour) + time.Minute * time.Duration(minute) + time.Second * time.Duration(second))
+	inc := startUnix.Add(time.Hour*time.Duration(hour) + time.Minute*time.Duration(minute) + time.Second*time.Duration(second))
```

Comment:
NP: can leave these unchanged

Reply:
Ya my editor did this, I'll change it back 👍